sklearn/feature_selection/_sequential.py

```diff
@@ -36,6 +36,11 @@ class SequentialFeatureSelector(SelectorMixin, MetaEstimatorMixin,
         to select. If float between 0 and 1, it is the fraction of features to
         select.
 
+    aborted_rate : float, default=None
```

Comment:
Please rename to `tol`.

Reply:
fix it on https://github.com/scikit-learn/scikit-learn/commit/44300288cc81e213203febe6eddaf928fcbb48f1#diff-521200bea206f673d4e933a33304f606f951d97edd7999b4d6b019cca79a81a0L39